src/argument.cpp

```diff
@@ -133,6 +133,12 @@ argument argument::reshape(const shape& s) const
     return {s, this->m_data};
 }
 
+argument argument::reshape_lazy(const shape& s) const
+{
+    assert(s.element_space() <= this->get_shape().element_space());
+    return {s, this->m_data};
+}
```

Comment:
This does the same thing as `reshape` method. It should probably be removed.

Reply:
Done